scst-store/install-scst-store.hbs.md

```diff
@@ -19,83 +20,117 @@ To install Supply Chain Security Tools - Store:
-      api_port                          9443                 integer  The internal port for the metadata app api endpoint. This will be used by the kube-rbac-proxy sidecar.
-      app_limit_cpu                     250m                 string   CPU limit for metadata-store-app container
-      app_replicas                      1                    integer  The number of replicas for the metadata-store-app
-      db_user                           metadata-store-user  string   The database user to create and use for updating and querying. The metadata postgres section create this user. The metadata api server uses this username to connect to the database.
-      pg_req_memory                     1Gi                  string   Memory request for postgres container in metadata-store-db deployment
-      priority_class_name                                    string   If specified, this value is the name of the desired PriorityClass for the metadata-store-db deployment
-      use_cert_manager                  true                 string   Cert manager is required to be installed to use this flag. When true, this creates certificates object to be signed by cert manager for the API server and Postgres database. If false, the certificate object have to be provided by the user.
-      api_host                          localhost            string   The internal hostname for the metadata api endpoint. This will be used by the kube-rbac-proxy sidecar.
-      db_password                       <auto-generated>     string   The database user password. If not specified, the password will be auto-generated.
-      storage_class_name                                     string   The storage class name of the persistent volume used by Postgres database for storing data. The default value will use the default class name defined on the cluster.
-      database_request_storage          10Gi                 string   The storage requested of the persistent volume used by Postgres database for storing data.
-      add_default_rw_service_account    true                 string   Adds a read-write service account which can be used to obtain access token to use metadata-store CLI
-      log_level                         default              string   Sets the log level. This can be set to "minimum", "less", "default", "more", "debug" or "trace". "minimum" currently does not output logs. "less" outputs log configuration options only. "default" and "more" outputs API endpoint access information. "debug" and "trace" outputs extended API endpoint access information(such as body payload) and other debug information.
+      KEY                             DEFAULT                                                                                                                                                                                                      TYPE     DESCRIPTION
```

Comment:
I think the snippet below is what the main change for this PR was supposed to be